src/components/FileMenu/utils.js

```diff
@@ -45,11 +49,76 @@ export const appPathFor = (fileType, id, apiVersion) => {
     }
 }
 
-export const preparePayloadForSaveAs = ({ ...visualization }) => {
+export const preparePayloadForSaveAs = ({
+    visualization,
+    name,
+    description,
+}) => {
     delete visualization.id
     delete visualization.created
     delete visualization.createdBy
     delete visualization.user
 
```

Comment:
Not really part of the scope but I think we should also delete subscribers when copying:
```suggestion
delete visualization.subscribers
```